src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.module.css

```diff
@@ -22,7 +22,7 @@
 
 .geneSymbol {
   font-weight: var(--font-weight-bold);
-  margin-right: 10px;
+  margin-right: 12px;
 }
 
```

Comment:
looks like the left margin here is off:

<img width="336" alt="image" src="https://github.com/Ensembl/ensembl-client/assets/6834224/0b0b1ba5-6c68-452e-a512-e821f475eb3b">

XD: 12 + 12; us: 12 + 15.

Reply:
Something-something line height. Notice how the downward strokes of letters in the biotype get cropped:

<img width="289" alt="image" src="https://github.com/Ensembl/ensembl-client/assets/6834224/51bb366f-e9b4-4c83-a85e-afac51712902">